src/server/visibility/registry.rs

```diff
@@ -3,82 +3,260 @@ use bevy::{
     utils::{TypeIdMap, TypeIdMapExt},
 };
 
-/// Maps filter types to bit indices (0-31).
+use super::{FilterScope, filters_mask::FilterBit};
+use crate::{
+    prelude::*,
+    shared::replication::registry::{ReplicationRegistry, component_mask::ComponentMask},
+};
+
+/// Maps visibility filters data to [`FilterBit`]s.
```

Comment:
```suggestion
/// Maps visibility filters data to [`FilterBits`](FilterBit).
```
I've found that separating out the 's' looks ugly.

Reply:
I agree, but I think using a different name might be a bit confusing. I pushed a rephrase and used the singular form.